app/routes.js

```diff
@@ -136,12 +136,27 @@ router.get('/checkAnswersPdf', (req, res) => {
             res.setHeader('Content-disposition', 'attachment; filename=checkYourAnswers.pdf');
             res.send(result);
         })
+        .catch(err => {
+            res.status(500).render('errors/500', {cmmon: commonContent});
+        });
+    ;
+});
+
+router.get('/declarationPdf', (req, res) => {
```

Comment:
Would prefer this route to use lowercase with hyphens, for example `/declaration-pdf`

Reply:
ok, this is done and now resides within documentDownload.js

Reply:
ok, this has been done and now resides within documentDownload.js